### ISSUE TITLE
Newest release unable to read VCF files 

### ISSUE THREAD

**Issue by cbossu** [thierrygosselin/radiator#86]
Hello,

I've run into a similar issue to others, where radiator cannot read a vcf file (while a previous version was able to). 

I use the following code:

dat<-tidy_genomic_data(data="HETH.all.clean.hethv0.rm_hets.rmrel.rmcont.filt5ClimGroup.vcf",strata="HETH.all.clean.hethv0.rm_hets.rmrel.rmcont.filt5ClimGroup19fix.ped.strat.txt")

The following is printed to the screen, and the error I receive is:
Reading VCF
Data summary: 
    number of samples: 164
    number of markers: 90439
done! timing: 30 sec

Filter monomorphic markers
Number of individuals / strata / chrom / locus / SNP:
    Blacklisted: 0 / 0 / 0 / 0 / 0

Filter common markers:
Number of individuals / strata / chrom / locus / SNP:
    Blacklisted: 0 / 0 / 1 / 9 / 9
Generating individual stats...
**Error: Tibble columns must have compatible sizes.
* Size 2: Column `READ_DEPTH`.
* Size 14830520: Columns `ALLELE_REF_DEPTH` and `ALLELE_ALT_DEPTH`.**
ℹ Only values of size one are recycled.
Run `rlang::last_error()` to see where the error occurred.

I have installed the newest version of radiator (1.1.6), but this error persists. I noticed you closed a similar thread a week ago, and I don't want to duplicate that post, but I don't know how to fix this error. I can send the vcf and strata file if you'd like. Best,

C

**Comment by thierrygosselin** [comment]
Hi sorry about the bug
What’s the version for these 2 packages on your computer:

- SeqArray
- dplyr 

Thank you
Thierry

**Comment by cbossu** [comment]
Hello,
	SeqArray 1.26.2 
	dplyr 1.0.0

Christen

Christen Bossu
The Institute of the Environment and Sustainability 
Center for Tropical Research
University of California, Los Angeles
Los Angeles, CA 90095-1496
cbossu@ucla.edu

**Comment by thierrygosselin** [comment]
Try updating SeqArray to 1.28.1
It’s bioconductor package
Normally reinstalling radiator should’ve installed that latest version but it’s in the suggest field ... might have to do something else 🤔

**Comment by cbossu** [comment]
Hi Thierry,
	Updating SeqArray seemed to work! Thank you! Just to note, you can only get SeqArray v 1.28.1 from GitHub as it’s the development version. When I installed from Bioconductor, it was still the earlier 1.26.2 version. 

if (!require("devtools")) install.packages("devtools")
devtools::install_github("zhengxwen/gdsfmt")
devtools::install_github("zhengxwen/SeqArray”)

	
Right now, it’s still tidying up the vcf file, but so far it made it a lot further than before, so I’m confident I’m okay. If another error pops up, I’ll let you know. Thanks again!

Christen

Christen Bossu
The Institute of the Environment and Sustainability 
Center for Tropical Research
University of California, Los Angeles
Los Angeles, CA 90095-1496
cbossu@ucla.edu

**Comment by thierrygosselin** [comment]
Ok it’s good to know
I’ll figure something out for the next push
Re-open this issue if you’re still experiencing problems

**Comment by cbossu** [comment]
Hello again,
	Your solution worked to read in my data. However, an error occurred when I was running the pairwise FST with confidence interval that I feel like leads back to the input data because the error relates to the MARKERS column. I checked the read in data, and it does have a MARKERS column, all with different names.

The command I ran was:

heth_str.fst.ci<-fst_WC84(dat,snprelate=F,strata=str,pairwise=TRUE,ci=TRUE,iteration.ci=100,verbose=TRUE,digits=2,pop.levels=c("AZ1","UT1","Yosemite","CA1","CA2","HaidaGwaii","AK2","PrinceRupert","Exstew","Kispiox","Whistler","Golden","Maxan","FortFraser","MacKenzie","Hudson'sHope","Swan","PA1","ME2"),filename="PairwiseFST",parallel.core=4)


And the error I got was:

Execution date/time: 20200703@1058
Folder created: PairwiseFST_fst_WC84_20200703@1058
Importing data
Analyzing strata file
    Number of strata: 19
    Number of individuals: 164
Global fst calculation
New names:
* MARKERS -> MARKERS...1
* MARKERS -> MARKERS...3
Error: Can't subset columns that don't exist.
x Column `MARKERS1` doesn't exist.
Run `rlang::last_error()` to see where the error occurred.
In addition: Warning messages:
1: Unknown or uninitialised column: `MARKERS`. 
2: Unknown or uninitialised column: `MARKERS1`. 

It looks like the error occurs when trying to change the MARKERS names? My marker names looks like the following from the tidied vcf file:  scaf_16__1__83120 , scaf_16__2__162968, scaf_16__3__163043, etc. 

Thanks,

Christen


Christen Bossu
The Institute of the Environment and Sustainability 
Center for Tropical Research
University of California, Los Angeles
Los Angeles, CA 90095-1496
cbossu@ucla.edu
